app/src/main/java/nl/eduvpn/app/service/SerializerService.java

```diff
@@ -134,13 +144,74 @@ public JSONObject serializeProfileList(@NonNull List<Profile> profileList) throw
         }
     }
 
+    public JSONObject serializeCurrentVPN(CurrentVPN currentVPN) throws UnknownFormatException {
+        JSONObject result = new JSONObject();
+        try {
+            if (currentVPN instanceof OpenVPN) {
+                result.put("vpn_type", "OpenVPN");
+                result.put("vpn_value", serializeProfile(((OpenVPN) currentVPN).getProfile()));
+            } else if (currentVPN instanceof WireGuard) {
+                result.put("vpn_type", "WireGuard");
+                result.put("vpn_value", serializeWireGuardConfig(((WireGuard) currentVPN).getConfig()));
+            } else {
+                //todo: remove this case by converting this file to Kotlin
+                throw new RuntimeException("Unknown current vpn when serializing.");
+            }
+        } catch (JSONException ex) {
+            throw new UnknownFormatException(ex);
+        }
+        return result;
+    }
+
+    public CurrentVPN deserializeCurrentVPN(JSONObject jsonObject) throws UnknownFormatException{
+        try {
+            String type = jsonObject.getString("vpn_type");
+            if(type.equals("OpenVPN")) {
```

Comment:
Space missing after `if` in this class as well. Maybe you need to change your code style?

Reply:
Fixed in https://github.com/eduvpn/android/pull/318/commits/3cd59bcbed42a20e85ff8b0b2a17b5afee371bab